uberfire-runtime-plugins/uberfire-runtime-plugins-backend/src/main/java/org/uberfire/ext/plugin/backend/PluginServicesImpl.java

```diff
@@ -615,44 +610,41 @@ public DynamicMenu getDynamicMenuContent( org.uberfire.backend.vfs.Path path ) {
                                 loadMenuItems( pluginName ) );
     }
 
+//    protected PerspectiveEditor makeDefaultPerspectiveEditor( final String pluginName ) {
```

Comment:
Any reason you prefer to add commented code blocks rather than delete (as it's retained in the version history)? ;)